pickle-processor/src/main/java/com/fourlastor/pickle/ScenarioOutlineConverter.kt

```diff
@@ -0,0 +1,39 @@
+package com.fourlastor.pickle
+
+import gherkin.ast.Examples
+import gherkin.ast.Scenario
+import gherkin.ast.ScenarioOutline
+import gherkin.ast.Step
+import gherkin.ast.TableCell
+import gherkin.ast.TableRow
+
+internal fun Examples.toScenarios(outline: ScenarioOutline): List<Scenario> {
+    return tableBody.map { example ->
+        outline.createExampleScenario(tableHeader, example)
+    }
+}
+
+private fun ScenarioOutline.createExampleScenario(header: TableRow, example: TableRow): Scenario {
+    return Scenario(
```

Comment:
quick return? I should put a linter

Reply:
The project uses old Kotlin style which has 8 spaces indentation. I did quick return but it looked kind of ugly. 

We should migrate to official style guide with 4 spaces.

Reply:
yeah, I'll change this in the future!